internal/driver/driver.go

```diff
@@ -375,9 +315,15 @@ func (d *Driver) handleWriteCommands(devName string, p protocolMap, reqs []dsMod
 		if err != nil {
 			return 0, err
 		}
+
+		value, ok := v.(string)
```

Comment:
I'm not sure if I mentioned it before, but you can switch on the type of v to handle non-strings. I think I was getting float64 that I had to convert to uint64.

Reply:
HUmm, I thought we resolved this one already... Will look at it again, but may be a different location.

Reply:
_I am not saying there is anything wrong with the way the code works_, but I wanted to mention that it is possible to support numbers in addition to strings here.

I believe this comes from the Custom Impinj commands:
```yaml
    attributes:
      vendor: "25882"
      subtype: "23"
```

which are actually `uint` values and not strings. we could re-write this as:
```yaml
    attributes:
      vendor: 25882
      subtype: 23
```
the code would just need to be adapted to handle converting from `float64` to `uint64` since `float64` is the default number type I believe.

```go
switch value := v.(type) {
case uint64:
	return value, nil
case float64:
	return uint64(value), nil
case int64:
	return uint64(value), nil
case string:
	u, err := strconv.ParseUint(value, 10, 64)
	return u, errors.Wrapf(err, "unable to parse attribute %q with val %q as uint", key, value)
default:
	return 0, fmt.Errorf("unable to convert attribute %q with unsupported type %T to uint", key, value)
}
```
of course if we know for sure the numbers are only ever sent as `float64` we can remove the `uint64` and `int64` cases and just have 2:

```go
switch value := v.(type) {
case float64:
	// number values are always float64 and need to be cast to uint64
	return uint64(value), nil
case string:
	u, err := strconv.ParseUint(value, 10, 64)
	return u, errors.Wrapf(err, "unable to parse attribute %q with val %q as uint", key, value)
default:
	return 0, fmt.Errorf("unable to convert attribute %q with unsupported type %T to uint", key, value)
}
```

Reply:
Right, we had attempted using numbers and reverted as it was causing issues and was a very low priority refactoring compared to the rest of this PR. ;-)

Reply:
leaving this as is for now